orderer/consensus/etcdraft/membership.go

```diff
@@ -112,94 +105,6 @@ func ComputeMembershipChanges(oldMetadata *etcdraft.BlockMetadata, oldConsenters
 	return result, nil
 }
 
-func validateConsenterTLSCerts(c *etcdraft.Consenter, ordererConfig channelconfig.Orderer) error {
-	clientCert, err := parseCertificateFromBytes(c.ClientTlsCert)
-	if err != nil {
-		return errors.Wrap(err, "parsing tls client cert")
-	}
-	serverCert, err := parseCertificateFromBytes(c.ServerTlsCert)
-	if err != nil {
-		return errors.Wrap(err, "parsing tls server cert")
-	}
-
-	opts, err := createX509VerifyOptions(ordererConfig.Organizations())
-	if err != nil {
-		return errors.WithMessage(err, "creating x509 verify options")
-	}
-	_, err = clientCert.Verify(opts)
-	if err != nil {
-		return fmt.Errorf("verifying tls client cert with serial number %d: %v", clientCert.SerialNumber, err)
-	}
-
-	_, err = serverCert.Verify(opts)
-	if err != nil {
-		return fmt.Errorf("verifying tls server cert with serial number %d: %v", serverCert.SerialNumber, err)
-	}
-
-	return nil
-}
-
-func createX509VerifyOptions(orgs map[string]channelconfig.OrdererOrg) (x509.VerifyOptions, error) {
```

Comment:
why did you move these to a different file?

Reply:
membership has nothing to do with config validation. Those functions should be near VerifyConfigMetadata.

Reply:
I agree that `ComputeMembershipChanges` should not be responsible for TLS validations.  However, for future reference, when performing operations like moving code which produces a large diff, it's much better to perform the move in its own PR, or minimally, in its own commit.  It makes reviewing changes much easier.

Reply:
sorry for that, will take into account next time :)